src/test/java/de/bsi/secvisogram/csaf_cms_backend/model/template/TemplateServiceTest.java

```diff
@@ -0,0 +1,55 @@
+package de.bsi.secvisogram.csaf_cms_backend.model.template;
```

Comment:
There's a discrepancy in naming. The files in the main package all are called `DocumentTemplate...` whereas in the tests the `Document...` part is skipped. I plead in favor of consistency to remove the `Document` prefix altogether.

Reply:
Ok